Dockerfile

```diff
@@ -53,6 +53,7 @@ RUN apt-get update \
     make \
     gcc \
     git \
+    pkg-config \
```

Comment:
The module started using pkg-config to find C headers and libraries, as covered in changelog.